sympy/polys/ring_series.py

```diff
@@ -101,18 +93,19 @@ def rs_is_puiseux(p, x):
     >>> rs_is_puiseux(p, x)
     True
     """
-    ii = p.ring.gens.index(x)
-    n = 1
+    index = p.ring.gens.index(x)
     for k in p:
-        if k[ii] != int(k[ii]):
+        if k[index] != int(k[index]):
             return True
-        if k[ii] < 0:
+        if k[index] < 0:
```

Comment:
BTW renaming `ii` to `index` is pointless.
Also, funny as it may sound, `index` isn't a good name for an index in code, idiomatic (and instantly understood) names for index variables are `i`, `j`, `k` etc. `index` carries a hint that the code is talking about an index in the application domain. Since we do have index as a domain term (Einstein indexes and such), this is actually actively confusing, so I recommend renaming `ii` to `i` (unless `i` is already taken).